docs/faq.md

```diff
@@ -32,19 +32,73 @@ See the Semgrep feature and pricing comparison at [r2c.dev/pricing](https://r2c.
 For the Semgrep Team edition, a user is any project contributor whose pull request or code was scanned in the billing cycle. For example, if a project has 4 unique contributors who pushed code during the period, and Semgrep runs on all their pull requests, there are 4 users. If these unique contributors push to many projects, they will only be counted once; please feel free to scan often and widely. Semgrep does not charge on a line-of-code or per-project basis. 
 
 # Comparisons
+### How is Semgrep different than $OTHER_TOOL or $GENERIC_[SAST](https://en.wikipedia.org/wiki/Static_application_security_testing)?
 
-#### How is Semgrep different than linters?
-Linters use static analysis but have a narrower scope for analysis (most rules typically operate on a single line).
+Semgrep is an open-source tool with a simple syntax for writing rules: if you can write code, you can write a Semgrep rule — no program analysis PhD required!
 
-Semgrep incorporates rulesets inspired by the rules of many popular linters and checkers, including ESLint, RuboCop, Bandit, and FindSecBugs. But Semgrep also allows you to enable multiple rulesets at the same time without adding linter-specific artifacts or installation to your code repository.
+To our knowledge, the only other tool with the explicit goal of allowing custom rules is Semmle’s proprietary tool, CodeQL. CodeQL has a domain-specific language which is extremely powerful but is designed for those with significant program analysis expertise, whereas Semgrep is designed for the security engineer* ** *or developer who wants to automate code review. Our goal is to make writing a Semgrep rule as easy as copying the code you want to find—and letting the Semgrep engine to make the rule and autofix high-quality enough to run in CI or the editor.
 
-Lastly, while many linters are extensible, with Semgrep you don’t need to learn specific abstract syntax tree (AST) based patterns for writing custom rules. You can read more about the spectrum from linters to complex analysis tools in Instagram's wonderful article [Static Analysis at Scale: An Instagram Story](https://instagram-engineering.com/static-analysis-at-scale-an-instagram-story-8f498ab71a0c).
+Our hosted offering, [Semgrep.dev](http://semgrep.dev/), has a generous free tier (even for private repos!) that offers a hosted CI integration with one-click setup and diff-awareness so you can start running Semgrep right away on new code so that you don’t have to fix all the existing issues when you first start out. For users running inside orgs with lots of repos, the hosted offering also offers a policy and notification system that makes it easy to tune Semgrep so that it only reports issues or suggest fixes that actually get applied. Our goal is a 99% fix rate for what Semgrep reports.
```

Comment:
I suggest breaking up the first sentence into two or three sentences. Something like:
```suggestion
Our hosted offering, [Semgrep.dev](http://semgrep.dev/), has a generous free tier (even for private repos!). If offers a hosted CI integration with one-click setup so you can start running Semgrep right away. Its diff-awareness lets you scan new code and doesn’t force you to fix all the existing issues when you first start out. For users running inside orgs with lots of repos, the hosted offering also offers a policy and notification system that makes it easy to tune Semgrep so that it only reports issues or suggest fixes that actually get applied. Our goal is a 99% fix rate for what Semgrep reports.
```